controllers/event_controller.py

```diff
@@ -170,6 +171,21 @@ def _render(self, event_key):
         if event_insights:
             event_insights_template = 'event_partials/event_insights_{}.html'.format(event.year)
 
+        # rankings processing for ranking score per match
+        full_rankings = event.rankings
+        rankings_enhanced = event.rankings_enhanced
+        if rankings_enhanced is not None:
```

Comment:
add `and event.year in RankingIndexes.CUMULATIVE_RANKING_SCORE` so we don't get crashes when 2017 events are added before the constants are updated